### PR TITLE
Properly escape JSON strings

### PR DESCRIPTION
**Pull request by dbwiddis**

### CODE REVIEW

File: src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java [hunk 1]
```diff
@@ -587,16 +587,16 @@ public int getPhysicalProcessorCount() {
     @Override
     public String toJSON() {
         StringBuilder sb = new StringBuilder("{");
```
Comment (dblock) [inline src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java]: There's got to be a JSON parser where you can turn a Dictionary into JSON which takes care of all of this ...

Comment (dbwiddis) [inline src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java]: There are dozens. Jackson is the most common. 

The overhead to create the dictionaries is more lines of code than building the strings. And this way avoids another dependency.

Comment (dblock) [inline src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java]: Just think about it :)

Comment (dbwiddis) [inline src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java]: Already working on porting existing code over to `javax.json`. :)

**Pull request merged**